docs/ux_and_ui.md

```diff
@@ -0,0 +1,82 @@
+---
+id: developer-tools-ux-ui
+title: Our Favorite Tools for UX/UI
+sidebar_label: UX/UI Tools
+---
+
+## Figma
+
+### What is Figma?
+
+It is a collaborative interface, cloud-based design and prototyping tool for digital projects.
+
+### Why we use Figma?
+
+1. Works for individual projects as well as massive team endeavor.
+2. Automated tasks via plugins for repeated elements to speed up projects.
+3. Smart selection tool with auto adjustments for spacing, arrangement, and organizing.
+4. Flexible styles that save right to your project.
+5. Create design systems and components.
+6. Drag and drop accessible libraries.
+
+### Figma Installation
+
+[Figma Installation](https://www.figma.com/downloads/)
+
+<iframe
+width="100%" height="315" src="https://www.youtube.com/embed/Cx2dkpBxst8" frameborder="0" allowfullscreen="true"> </iframe>
+
+* * *
+
+## Zeplin
+
+### What is Zeplin?
+
+Zeplin is a plugin and a application for Mac and Windows specialized for helping in the transition between designers and engineers.
+
+### Why we use Zeplin?
+
+Zeplin facilitates the transfer of designs by taking them from Sketch, Adobe XD, Figma, and Photoshop and exporting them into a format that can easily generate code snippets, style guides, specs, and assets.
+
+### Zeplin Installation
+
+[Zeplin Installation](https://support.zeplin.io/en/articles/244698-downloading-mac-and-windows-apps)
+
+<iframe
+width="100%" height="315" src="https://www.youtube.com/embed/x1RPNx8Jsp4" frameborder="0" allowfullscreen="true"> </iframe>
+
+* * *
+
+## Sketch
+
+### What is Sketch?
+
+It is a graphic editor for macOS developed by Dutch Company.
+
+### Why we use Sketch?
+
+It is used for user experience and user interface in web and mobile app design. Some of the features that this technology has is the prototyping and collaboration in the design.
+
+### Sketch Installation?
+
+[Sketch Installation](https://www.sketch.com/updates/)
+
+ <iframe
+width="100%" height="315" src="https://www.youtube.com/embed/ilcwjXTqyNM"  frameborder="0" allowfullscreen="true"> </iframe>
+
+* * *
+
+## Material.io
+
+### What is Material.io
```

Comment:
Missing question mark, or well, in others titles, remove that one to make it consistent